### PR TITLE
[SOC-5187]Unexpected validation name when editing space name

### PR DESCRIPTION
**Pull request by AmeniJ**

### CODE REVIEW

File: component/webui/src/main/java/org/exoplatform/social/webui/space/UISpaceSettings.java [hunk 1]
```diff
@@ -48,7 +48,7 @@ public UISpaceSettings(String name) throws Exception {
     //
     addUIFormInput(new UIFormStringInput(SPACE_DISPLAY_NAME, SPACE_DISPLAY_NAME, "").
                    addValidator(MandatoryValidator.class).
-                   addValidator(ExpressionValidator.class, "^([\\p{L}\\s\\d\']+[\\s]?)+$", MSG_INVALID_SPACE_NAME).
+                   addValidator(ExpressionValidator.class, "^([\\p{L}\\d\']+[\\s]?)+$", MSG_INVALID_SPACE_NAME).
```
Comment (thomasdelhomenie) [inline component/webui/src/main/java/org/exoplatform/social/webui/space/UISpaceSettings.java:51]: This change aligns the regex with the one in UISpaceInfo, so it is a good thing, but I am not sure to understand it completely. It seems more complex than necessary. We allow only letters, digits, spaces and single quotes, so why not simply "^[\p{L}\d\s']+$" ?

Comment (AmeniJ) [inline component/webui/src/main/java/org/exoplatform/social/webui/space/UISpaceSettings.java:51]: I tested this regex "^[\p{L}\d\s']+$" but the problem is not solved (the name with 2 spaces is accepted at creation).

Comment (thomasdelhomenie) [inline component/webui/src/main/java/org/exoplatform/social/webui/space/UISpaceSettings.java:51]: Is it forbidden to have 2 spaces ?

Comment (AmeniJ) [inline component/webui/src/main/java/org/exoplatform/social/webui/space/UISpaceSettings.java:51]: Yes, it is forbidden to have 2 spaces.

Comment (thomasdelhomenie) [inline component/webui/src/main/java/org/exoplatform/social/webui/space/UISpaceSettings.java:51]: Ah :) Where did you see this rule ?

Comment (AmeniJ) [inline component/webui/src/main/java/org/exoplatform/social/webui/space/UISpaceSettings.java:51]: The name with two spaces is not accepted at edition so  there is no sense to accept it at creation.

Comment (thomasdelhomenie) [inline component/webui/src/main/java/org/exoplatform/social/webui/space/UISpaceSettings.java:51]: So the problem is probably at edition. It should be allowed at creation and edition IMO.

**Pull request closed without merging**